odoo-shippable/Dockerfile

```diff
@@ -1,145 +1,13 @@
-
-# Install depends
-RUN apt-get remove -y git \
-    && apt-get update \
-    && apt-get install -y software-properties-common \
-    && add-apt-repository ppa:git-core/ppa -y \
-    && apt-get update \
-    && apt-get install -y xsltproc xmlstarlet openssl \
-        poppler-utils antiword p7zip-full expect-dev mosh bpython \
-        ghostscript graphviz openssh-server zsh \
-        lua50 liblua50-dev liblualib50-dev \
-        exuberant-ctags python-dev gcc g++ libtiff5-dev libjpeg8-dev zlib1g-dev \
-        libfreetype6-dev liblcms2-dev libwebp-dev \
-        git \
-    && pip install --upgrade SOAPpy pyopenssl suds \
-        pillow qrcode xmltodict M2Crypto \
-        recaptcha-client egenix-mx-base \
-        PyWebDAV mygengo pandas==0.16.2 numexpr==2.4.4 \
-        ndg-httpsclient pyasn1 \
-        watchdog isort coveralls line-profiler \
-    && echo "PermitRootLogin yes" >> /etc/ssh/sshd_config \
-    && curl -fsSL https://raw.github.com/robbyrussell/oh-my-zsh/master/tools/install.sh | sh; echo "$?" \
-    && sed -i 's/robbyrussell/odoo-shippable/g' /root/.zshrc \
-    && curl https://gist.githubusercontent.com/schminitz/9931af23bbb59e772eec/raw/cb524246fc93df242696bc3f502cababb03472ec/schminitz.zsh-theme -o /root/.oh-my-zsh/themes/odoo-shippable.zsh-theme \
-    && rm -rf /var/lib/apt/lists/*
+COPY scripts/entrypoint-image.py /entrypoint_image
+COPY scripts/*.sh /usr/share/vx-docker-internal/odoo-shippable/
+RUN bash /usr/share/vx-docker-internal/odoo-shippable/build-image.sh
 
-# Fix reinstalling npm v3 fails on docker https://github.com/npm/npm/issues/9863
-#  And install localtunnel similar to ngrok
-RUN cd $(npm root -g)/npm \
-  && npm install fs-extra \
-  && sed -i -e s/graceful-fs/fs-extra/ -e s/fs.rename/fs.move/ ./lib/utils/rename.js \
-  && npm install -g localtunnel
-
-# Install hub, git prompt, git auto-completion, other dev packages
-# Install spf13 for vim, fix comments hidden, fix select text
-#  and disable neocomplete at startup
-# Add compatibility with "travis_wait" command.
-RUN wget https://github.com/github/hub/releases/download/v2.2.2/hub-linux-amd64-2.2.2.tgz -O /tmp/hub.tgz \
-  && tar -xvzf /tmp/hub.tgz -C /tmp/ \
-  && cp /tmp/hub*/bin/hub /usr/bin/hub \
-  && rm -rf /tmp/hub*/ && rm /tmp/hub.tgz \
-  && wget https://raw.githubusercontent.com/github/hub/master/etc/hub.bash_completion.sh -O /usr/local/bin/hub.bash_completion.sh \
-  && echo "if [ -f /usr/local/bin/hub.bash_completion.sh   ]; then . /usr/local/bin/hub.bash_completion.sh; fi" >> ${HOME}/.profile \
-  && curl https://raw.githubusercontent.com/git/git/master/contrib/completion/git-prompt.sh -o /usr/local/bin/git-prompt.sh \
-  && echo 'source /usr/local/bin/git-prompt.sh\nsource ${HOME}/.bash_colors' >> ${HOME}/.profile \
-  && curl https://raw.githubusercontent.com/git/git/master/contrib/completion/git-completion.bash -o /usr/local/bin/git-completion.bash \
-  && echo 'source /usr/local/bin/git-completion.bash' >> ${HOME}/.profile \
```

Comment:
`git-prompt.sh` is shipped with the git package and can be found at `/etc/bash_completion.d/git-prompt`
`git-completion.bash` is shipped with the git package and can be found at `/usr/share/bash-completion/completions/git`